### DISCUSSION TITLE
Q: Support for building as a  "component" or "sub-app" to transition existing apps

### DISCUSSION THREAD

**Discussion by kestred** [mint-lang/mint#163] [Q&A]
Supposing that there is an existing app (such as a React or Vue app), is there a way to build a project as either single components or "sub-applications" that could be wrapped in JavaScript by an existing application? (for the normal goal of "incremental migration")

Is this an objective the project is open to? If so, what direction towards implementing it might be welcomed as a pull request?

**Comment by gdotdesign** [comment]
It is already possible to to that, it's just not documented yet :arrow_right: https://github.com/mint-lang/mint/issues/144#issuecomment-559458138

I'll keep this issue open for the documentation.
